components/eventbrite/sources/new-event-ended/new-event-ended.js

```diff
@@ -0,0 +1,59 @@
+const common = require("../common.js");
+
+module.exports = {
+  ...common,
+  key: "eventbrite-new-event-ended",
+  name: "New Event Ended",
+  description: "Emits an event when an event has ended",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    ...common.props,
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+  },
+  hooks: {
```

Comment:
Remember to include the parent's hooks:

```suggestion
  hooks: {
    ...common.hooks,
```

Reply:
Updated